docs/encyclopedia/workflows.mdx

```diff
@@ -626,7 +626,11 @@ The Workflow Id Reuse Policy can have one of the following values:
 - **Terminate if Running:** Specifies that if a Workflow Execution with the same Workflow Id is already running, it should be terminated and a new Workflow Execution with the same Workflow Id should be started. This policy allows for only one Workflow Execution with a specific Workflow Id to be running at any given time.
 
 The first three values (Allow Duplicate, Allow Duplicate Failed Only, and Reject Duplicate) of the Workflow Id Reuse Policy apply to Closed Workflow Executions that are retained within the Namespace.
-For example, the Temporal Service can only check the Workflow Id of the spawning Workflow Execution based on the Workflow Id Reuse Policy against the Closed Workflow Executions for the last 30 days given a default Retention Period.
+For example, the Temporal Service can only check the Workflow Id of the spawning Workflow Execution based on the Workflow Id Reuse Policy against the Closed Workflow Executions for the last *30 days* given a default Retention Period. 
+
+If you need to start a Workflow for a particular implementation only if it hasn't started yet, pay close attention to your Retention Period.
```

Comment:
I find this line completely confusing. Pay close attention to what in the retention period? Why? What does the retention period tell you? How would you respond? When would you do this? Why? 

Would combining this line with the next one make this less confusing? Who wants to start Workflows atomically, and why are they trying to do this?

Reply:
Yeah, good points, I'll go into more detail here.